### PR TITLE
https://github.com/apache/dubbo-awesome/issues/31

### PR DESCRIPTION
**Pull request by fanla2021**
完成nacos注册中心文档优化

### CODE REVIEW

File: content/zh/docs3-building/docs/references/registry/nacos.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+当Dubbo使用`3.0.0`及以上版本时，需要使用Nacos `2.0.0`及以上版本。
+## 快速上手
+
+Dubbo 融合 Nacos 成为注册中心的操作步骤非常简单，大致步骤可分为“增加 Maven 依赖”以及“配置注册中心“。
+
+
+### 增加 Maven 依赖
+
+首先，您需要将 `dubbo-registry-nacos` 的 Maven 依赖添加到您的项目 `pom.xml` 文件中，并且强烈地推荐您使用 Dubbo `2.6.5`：
```
Comment (wangchengming666) [inline content/zh/docs3-building/docs/references/registry/nacos.md:23]: why use Dubbo `2.6.5`

**Review by wangchengming666 — CHANGES_REQUESTED** [review]

File: content/zh/docs3-building/docs/references/registry/nacos.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
+---
+type: docs
+title: "Nacos 注册中心"
+linkTitle: "Nacos"
+weight: 1
+description: "Nacos 注册中心参考手册"
+---
+
+Nacos 是 Dubbo 生态系统中重要的注册中心实现，其中 [`dubbo-registry-nacos`](https://github.com/apache/incubator-dubbo/tree/master/dubbo-registry/dubbo-registry-nacos) 则是 Dubbo 融合 Nacos 注册中心的实现。
```
Comment (AlbumenJ) [inline content/zh/docs3-building/docs/references/registry/nacos.md:9]: use `https://github.com/apache/dubbo/tree/3.0/dubbo-registry/dubbo-registry-nacos`

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

File: content/zh/docs3-building/docs/references/registry/nacos.md (hunk 1 above)
Comment (fanla2021) [reply content/zh/docs3-building/docs/references/registry/nacos.md]: Delete Dubbo 2.6.5

File: content/zh/docs3-building/docs/references/registry/nacos.md (hunk 2 above)
Comment (fanla2021) [reply content/zh/docs3-building/docs/references/registry/nacos.md]: modify the link address of Dubbo registry Nacos

**Pull request merged**